app/controllers/api/v1/slides_controller.rb

```diff
@@ -22,7 +22,7 @@
 #
 module Api
   module V1
-    class SlidesController < ApplicationController
+    class SlidesController < Api::V1::BaseController
```

Comment:
__[Reek]__

[InstanceVariableAssumption] Api::V1::SlidesController assumes too much for instance variable '@slides'


<!-- Comment by [SideCI](https://sideci.com) -->